azure-pipelines.yml

```diff
@@ -0,0 +1,40 @@
+# https://aka.ms/yaml
+
+# Only for coverage!
+trigger:
+  paths:
+    include:
+    - 'src/*'
+    - 'CMakeLists.txt'
+    - 'azure-pipelines.yml'
+
+jobs:
+- job: test
+  displayName: 'Building and Testing'
+  strategy:
+    matrix:
+      Linux:
+        vmImage: ubuntu-18.04
+  pool:
+    vmImage: $(vmImage)
+
+  # https://github.com/dutor/nebula-gears/issues/1
+  container: shylockhg/nebula-dev:centos7
```

Comment:
Why not use the image `vesoft/nebula-dev:centos7` ?

Reply:
It's the previous image, the newer compiler can't work well for coverage, the detail See the issue in comment.